src/main/java/xyz/parkh/doing/domain/entity/AuthVo.java

```diff
@@ -0,0 +1,16 @@
+package xyz.parkh.doing.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
```

Comment:
JPA 엔티티를 제외하고는 가능하면 객체의 멤버 변수는 final 로 지정하여 불변하게 만들어주는 것이 좋습니다.
@AllArgsConstructor 어노테이션 사용은 지양하는게 좋을 것 같습니다. 
인스턴스 멤버 선언 순서에 영향을 받아 의도하지 않은 예외를 발생 시킬 수 있기 때문에 생성자를 명시적으로 만들고 생성자 레벨에서 @Builder 를 적용하는 것을 추천드립니다.
기본생성자는 접근 제어를 지정하여 아무 의미없는 객체 생성을 막는 것을 권장하고 있습니다.

@Data 어노테이션도 지양하는 것이 좋습니다. 
강력한 기능이긴 하지만 무분별한 setter 생성과 현재는 문제없지만 JPA 를 사용하게 될 경우 @ToString 으로 인한 순환 참조 문제를 야기 시킬 수 있어 가급적 필요한 어노테이션을 선택적으로 사용하는 것이 좋습니다.
setter 의 경우 필요한 경우에만 의미있는 이름으로 메소드를 새로 만들어 사용하는 것이 유지보수 차원에서 좋습니다.

그리고 가급적 멤버 변수는 immutable 하게 final 지정 해주시는 것이 좋습니다.
```suggestion
@Getter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
public class AuthVo {
    private final String userId;
    private final String password;
    
    @Builder
    public AuthVo(String userId, String password) {
        this.userId = userId;
        this.password = password;
    }
    
    public void updatePassword(String password) {
        this.password = password;
    }
}
```